### PR TITLE
No more free stuff in maint

### PR DESCRIPTION
**Pull request by spessbro**
## About The Pull Request
This pr removes some objects from maint on some maps that either don't make much sense being in maint like surgical drapes and stuff that makes maint free loot if maint loot needs to be buffed it should be through the helper not in maps.
-Putting this here instead of changelog to keep the pl*yers in the dark furthering coder superiority
<details>
<summary>Metastation</summary>

add:proper seed extractor to maint botany
add:cables into abandoned med to shock the grille akin to abandoned kitchen
add:seed vendor refill to maint botany
add:peacekeeper borg plushie in abandoned med
add:added chef's apron abandoned med
removed:growns from maint botany
removed:mutated seeds from maint botany
removed:one of the portaseeders from maint botany
removed:syringe gun in abandoned med
removed:cryoxadone beaker from abandoned bar
removed:regular apron from abandoned med
</details>
<details>
<summary>Deltastation</summary>

tweak:switched around tank holder and plants in atmos maint for better accessibility
add:kitchen knife to maint kitchen
add:budget insuls to maint robotics
add:random glowstick to maint robotics
add:empty machine frame to maint robotics
add:exosuit fab board to maint robotics
add:var edited meme organ spawner to abandoned library backroom with lootcount=1
add:plant in science maint
add:dice bag to gambling den
add:grille or trash spawner to maint robotics
add:tool closet in auxiliary engineering
add:tool closet in auxiliary atmos
add:sec bomb suit closet to where the maint nuke is
add:electrified grille into where the maint nuke is
add:regular deck of card to gambling den
add:gambling loot spawner to gambling den
add:money loot spawner to gambling den
add:two maint loot spawners to room next to maint robotics
tweak:changed some bookcases in abandoned library to spawners
removed:butcher knife from maint kitchen
removed:all money from gambling den
removed:cryoxadone beaker from maint bar
removed:welding helmet from maint robotics
removed:bone "tongue" from abandoned library backroom
removed:multitool in the room next to maint robotics
removed:wirerod from abandoned room next to sec with the waiter outfit
removed:cuffs from the room with owl costume
removed:exosuit fab from maint robotics
removed:seclite from maint robotics
removed:barricade in maint robotics
removed:emp grenade from abandoned storage (not sure what to call it)
removed:electrical supplies closet from auxiliary engineering
removed:electrical supplies closet from auxialiary atmos
removed:big sunglasses from gambling den
removed:regular sunglasses from gambling den
removed:officer closet from where the maint nuke is
removed:syndicate cards from gambling den
removed:insuls from auxiliary engineering
</details>
<details>
<summary>Icebox</summary>

add:bedsheet bin to maint surgery
removed:surgical drapes in maint surgery
</details>
<details>
<summary>Pubbystation</summary>

removed:surgical drapes in maint surgery
</details>

## Why It's Good For The Game
As an assistant main this hurts me more than anyone but the cool thing about maint is having stuff be randomized instead of knowing the meta of the map. These changes are not all removals such as the addition of a proper seed extractor to the meta botany maint.

## Changelog
:cl:PillowKakes
tweak: The rats have removed and added things to maintenance
/:cl:

### CODE REVIEW

**Comment by Rohesie** [comment]
Bold.
Could you document everything you've removed and everything you've added to each map?

**Comment by Pumpkinoe** [comment]
this hardly removes any free stuff
the surgery stuff is still there in that one place, you just removed the drapes
stuff like the syndie cards & sunglasses are still there too on delta
you have really mostly just removed the botany from maint :(

**Comment by spessbro** [comment]
> Could you document everything you've removed and everything you've added to each map?
Will do
> stuff like the syndie cards & sunglasses are still there too on delta
syndie cards are flavorful to have in maint and aren't that much of an issue, as for the sunglasses, maint is the only actual source of sunglasses for the bounty that i know of.

File: _maps/map_files/Deltastation/DeltaStation2.dmm [hunk 1]
```diff
@@ -69159,7 +69159,7 @@
 /area/maintenance/port)
 "cCF" = (
 /obj/effect/turf_decal/bot,
-/obj/structure/tank_holder/emergency_oxygen,
```
Comment (ATH1909) [inline _maps/map_files/Deltastation/DeltaStation2.dmm:69162]: ah, the emergency oxygen tank holder, truly a powerful maintenance item that people meta rush consistently

File: _maps/map_files/Deltastation/DeltaStation2.dmm [hunk 2]
```diff
@@ -4227,9 +4227,9 @@
 /obj/structure/rack,
 /obj/effect/decal/cleanable/dirt,
 /obj/item/storage/box/donkpockets,
-/obj/item/kitchen/knife/butcher,
```
Comment (ATH1909) [inline _maps/map_files/Deltastation/DeltaStation2.dmm:4230]: this is something that you can just print from a hacked autolathe, and which is equivalent in melee force to a circular saw, which can be printed from a not-hacked autolathe

File: _maps/map_files/Deltastation/DeltaStation2.dmm (hunk 1 above)
Comment (ATH1909) [reply _maps/map_files/Deltastation/DeltaStation2.dmm:69162]: oh, wait, you just swapped this thing's location with a plant, my bad

File: _maps/map_files/Deltastation/DeltaStation2.dmm (hunk 2 above)
Comment (spessbro) [reply _maps/map_files/Deltastation/DeltaStation2.dmm:4230]: yes

File: _maps/map_files/Deltastation/DeltaStation2.dmm (hunk 1 above)
Comment (spessbro) [reply _maps/map_files/Deltastation/DeltaStation2.dmm:69162]: it's replaced by plant, plants are nice

File: _maps/map_files/Deltastation/DeltaStation2.dmm [hunk 3]
```diff
@@ -87912,10 +87912,8 @@
 /area/crew_quarters/abandoned_gambling_den)
 "dob" = (
 /obj/structure/table/wood/poker,
-/obj/item/stack/spacecash/c1000{
```
Comment (ATH1909) [inline _maps/map_files/Deltastation/DeltaStation2.dmm:87915]: ... fair enough.

could you maybe replace this with a stack of a random amount of money, though? like, can't you just use a coin/space cash loot table or something?

File: _maps/map_files/Deltastation/DeltaStation2.dmm [hunk 4]
```diff
@@ -93463,7 +93461,6 @@
 "dBS" = (
 /obj/structure/table/wood/poker,
 /obj/item/reagent_containers/food/drinks/shaker,
-/obj/item/reagent_containers/glass/beaker/cryoxadone,
```
Comment (ATH1909) [inline _maps/map_files/Deltastation/DeltaStation2.dmm:93466]: there's usually at least one spare (as in, not being used in a cryo pod) cryo beaker out in the open in medbay near shiftstart, and I don't see people grab that most rounds; why does this maint one have to be removed?

Comment (spessbro) [reply _maps/map_files/Deltastation/DeltaStation2.dmm:93466]: doesn't make sense for one to be in bar

File: _maps/map_files/Deltastation/DeltaStation2.dmm [hunk 5]
```diff
@@ -93494,8 +93491,8 @@
 "dBX" = (
 /obj/structure/rack,
 /obj/item/storage/toolbox/mechanical,
-/obj/item/clothing/head/welding,
```
Comment (ATH1909) [inline _maps/map_files/Deltastation/DeltaStation2.dmm:93497]: my god, a welding helmet, an item that can _also_ be printed from an autolathe (or even just be bought from a youtool), and which logically could have been used in the construction/reconstruction of the station!

Comment (spessbro) [reply _maps/map_files/Deltastation/DeltaStation2.dmm:93497]: if it's so easy to get then buy it from a youtool or get one from cargo. Which they might deny making for more interaction.

File: _maps/map_files/IceBoxStation/IceBoxStation.dmm [hunk 6]
```diff
@@ -30396,7 +30397,6 @@
 /obj/structure/table/optable{
 	name = "Robotics Operating Table"
 	},
-/obj/item/surgical_drapes,
```
Comment (ATH1909) [inline _maps/map_files/IceBoxStation/IceBoxStation.dmm:30399]: this right here is the change in this PR that most offends me. why would you choose the remove the DRAPES here? they can be replaced entirely by a bedsheet from dorms (or even maintenance itself). literally all this change does is make it slightly more inconvenient to do surgery in maint robotics if you forgot to pick up a bedsheet while going through your powergamer checklist and make it much harder for players who don't know that you can use bedsheets in the place of surgical sheets to do actual surgery in maint robotics.

in short, you're actively _discouraging_ people from using maint robotics for actual maint surgery and such, and instead encouraging people to just treat it as a source of minor loot.

why would someone leave a pile of surgical tools here but not surgical drapes, anyway?

File: _maps/map_files/MetaStation/MetaStation.dmm [hunk 7]
```diff
@@ -4597,7 +4597,6 @@
 /obj/item/bikehorn/rubberducky,
 /obj/effect/spawner/lootdrop/maintenance/two,
 /obj/item/clothing/gloves/color/fyellow,
-/obj/item/grenade/empgrenade,
```
Comment (ATH1909) [inline _maps/map_files/MetaStation/MetaStation.dmm:4600]: ehhhhhhhhhh...

okay, I guess? you really should add this to the general maint loot pool if it isn't in there already, though.

File: _maps/map_files/MetaStation/MetaStation.dmm [hunk 8]
```diff
@@ -50100,7 +50105,7 @@
 "csa" = (
 /obj/structure/table,
 /obj/item/restraints/handcuffs/cable/white,
-/obj/item/gun/syringe,
```
Comment (ATH1909) [inline _maps/map_files/MetaStation/MetaStation.dmm:50103]: what I said for the maint EMP grenade also applies to the maint syringe gun, I think.

File: _maps/map_files/MetaStation/MetaStation.dmm [hunk 9]
```diff
@@ -57338,28 +57343,12 @@
...
-/obj/item/reagent_containers/food/snacks/grown/watermelon,
-/obj/item/reagent_containers/food/snacks/grown/citrus/orange,
-/obj/item/reagent_containers/food/snacks/grown/grapes,
-/obj/item/reagent_containers/food/snacks/grown/cocoapod,
-/obj/structure/rack,
-/obj/item/seeds/wheat,
-/obj/item/seeds/watermelon,
-/obj/item/seeds/watermelon,
-/obj/item/seeds/grape,
-/obj/item/seeds/glowshroom,
```
Comment (ATH1909) [inline _maps/map_files/MetaStation/MetaStation.dmm:57355]: really?

you know that the garden is a publicly accessible area, right?

File: _maps/map_files/MetaStation/MetaStation.dmm [hunk 10]
```diff
@@ -57338,28 +57343,12 @@
...
 /obj/effect/turf_decal/stripes/line,
-/obj/item/seeds/cannabis/rainbow,
```
Comment (ATH1909) [inline _maps/map_files/MetaStation/MetaStation.dmm:57357]: boooooooooooooooooo

File: _maps/map_files/MetaStation/MetaStation.dmm [hunk 11]
```diff
@@ -57957,7 +57945,7 @@
 /turf/open/floor/plating,
 /area/maintenance/starboard/aft)
 "cOB" = (
-/obj/item/seeds/sunflower/moonflower,
```
Comment (ATH1909) [inline _maps/map_files/MetaStation/MetaStation.dmm:57960]: boy oh boy I sure do hate those people who try to have fun in maintenance by growing moonshine

File: _maps/map_files/MetaStation/MetaStation.dmm [hunk 12]
```diff
@@ -57378,7 +57367,6 @@
 	},
 /obj/item/seeds/carrot,
 /obj/effect/turf_decal/stripes/line,
-/obj/item/seeds/cannabis/white,
```
Comment (ATH1909) [inline _maps/map_files/MetaStation/MetaStation.dmm:57381]: also booooooooooooooooooo

grinding for these things with RNG is a bitch without that one botany tech, and you know it. having much easier/less RNG-dependent access to the evolved weed variants was one of the things that made metastation's maint more distinct from the maintenance layouts of other stations, and it was a nice thing to be able to flex knowledge about.

okay, that was admittedly a much weaker argument than my other arguments, but I like maint weed, okay? I like it when people actually use the ghetto setups in maint for gimmicks and such instead of just ignoring them because they got nerfed into being practically just aesthetic!

File: _maps/map_files/Deltastation/DeltaStation2.dmm (hunk 2 above)
Comment (ATH1909) [reply _maps/map_files/Deltastation/DeltaStation2.dmm:4230]: okay then why are you removing it for being too strong

File: _maps/map_files/Deltastation/DeltaStation2.dmm (hunk 4 above)
Comment (ATH1909) [reply _maps/map_files/Deltastation/DeltaStation2.dmm:93466]: you can make certain drinks and synthmeat out of it, no?

File: _maps/map_files/Deltastation/DeltaStation2.dmm (hunk 5 above)
Comment (ATH1909) [reply _maps/map_files/Deltastation/DeltaStation2.dmm:93497]: saying "nah fuck off" to someone just because you can isn't really an "interaction"

Comment (ATH1909) [reply _maps/map_files/Deltastation/DeltaStation2.dmm:93497]: it's just like how the chemists spending 10 seconds to make mutagen for the botanists or saying "nah fuck off" to them for no reason wasn't really an interaction back when botanists relied on chemists for mutagen.

well, I mean, it did _technically_ result in interactions, they just happened to be violent and over-escalated ones.

Comment (ATH1909) [reply _maps/map_files/Deltastation/DeltaStation2.dmm:93497]: if one side doesn't really lose anything by letting the other side get what they want, it's really hard to get a good, justified conflict to happen- that's part of why friendly antags (who, say, happened to only roll steal and escape alive objectives) suck so much.

wew, that was a tangent, wasn't it?

File: _maps/map_files/MetaStation/MetaStation.dmm (hunk 11 above)
Comment (spessbro) [reply _maps/map_files/MetaStation/MetaStation.dmm:57960]: mutated plant it has no place as a static maint spawn

**Comment by ATH1909** [comment]
> as for the sunglasses, maint is the only actual source of sunglasses for the bounty that i know of.

do secglasses not count for this?

**Comment by Jared-Fogle** [comment]
> do secglasses not count for this?

if they dont theres the crafting recipe that cuts them up into normal glasses

File: _maps/map_files/IceBoxStation/IceBoxStation.dmm (hunk 6 above)
Comment (spessbro) [reply _maps/map_files/IceBoxStation/IceBoxStation.dmm:30399]: doing surgery in maint is supposed to be inconvenient

Comment (Jared-Fogle) [reply _maps/map_files/IceBoxStation/IceBoxStation.dmm:30399]: bedsheets are a direct upgrade from drapes because you can put them on your neck

File: _maps/map_files/MetaStation/MetaStation.dmm (hunk 12 above)
Comment (spessbro) [reply _maps/map_files/MetaStation/MetaStation.dmm:57381]: mutated plants are supposed to be grindy not to be handed out every round just cause it looks nice.

File: _maps/map_files/Deltastation/DeltaStation2.dmm (hunk 3 above)
Comment (Jared-Fogle) [reply _maps/map_files/Deltastation/DeltaStation2.dmm:87915]: don't we have the 10 credit ones just for this reason

Comment (ATH1909) [reply _maps/map_files/Deltastation/DeltaStation2.dmm:87915]: exactly, the 10 credit ones can pad out the pool, with low chances for higher-denomination cash stacks or something.

File: _maps/map_files/MetaStation/MetaStation.dmm (hunk 11 above)
Comment (ATH1909) [reply _maps/map_files/MetaStation/MetaStation.dmm:57960]: but it's a _garbage_ mutated plant.

also, geraniums and lilies are mutated plants, and don't they spawn in some chapels?

File: _maps/map_files/MetaStation/MetaStation.dmm (hunk 12 above)
Comment (ATH1909) [reply _maps/map_files/MetaStation/MetaStation.dmm:57381]: but the weed mutations are especially grindy because there are like 4 of them from a single plant.

**Comment by necromanceranne** [comment]
Delta's still got an absurd amount of free stuff. The free sec locker in maint, open for theft. The auxiliary engineering room has several insulated gloves, multitools and inducers behind normal walls. The thousands of credits in the abandoned bar as well as the multiple pairs of shades between it and the abandoned gambling den, along with some syndicate cards. And the abandoned arcade has a 2 tc weapon (though it's lacking riot darts, but given it's proximity to cargo, that's not at all any difficulty for a cargo tech or the QM to resolve).

There is quite a lot more free gamer gear in maint, and frankly, you're gonna have your work cut out for you if you want to genuinely attempt to audit that shit.

**Comment by Winterous** [comment]
On meta: I mean the syringe gun is kinda fair to remove I guess, but why the beaker of cryoxadone in the abandoned bar? How many times has that been someone's looting target?
Maybe the bartender has taken it to make doctor's delight a few times, that's it.

**Comment by brotherbeyondd** [comment]
There are some things you can't just make up for by adding them to the random maint loot pool, like the beaker of cryo. Delta definitely needs some loot nerfed but not all stuff is "free stuff" in maint as guess where tiders die most often?

**Comment by msgerbs** [comment]
Why no changelog?

**Comment by spessbro** [comment]
> Delta's still got an absurd amount of free stuff. The free sec locker in maint, open for theft. The auxiliary engineering room has several insulated gloves, multitools and inducers behind normal walls. The thousands of credits in the abandoned bar as well as the multiple pairs of shades between it and the abandoned gambling den, along with some syndicate cards. And the abandoned arcade has a 2 tc weapon (though it's lacking riot darts, but given it's proximity to cargo, that's not at all any difficulty for a cargo tech or the QM to resolve).
> 
> There is quite a lot more free gamer gear in maint, and frankly, you're gonna have your work cut out for you if you want to genuinely attempt to audit that shit.

will look at that

**Comment by spessbro** [comment]
> Why no changelog?

the players must be kept in the dark, also joking aside having a real changelog of every change would be horrible and not needed plus spoilers

**Comment by Tharcoonvagh** [comment]
Misleading pr title

**Comment by msgerbs** [comment]
> 
> 
> > Why no changelog?
> 
> the players must be kept in the dark, also joking aside having a real changelog of every change would be horrible and not needed plus spoilers

At least put something slightly more descriptive about rebalancing/adjusting maint loot. I'm not opposed to cutesy changelogs but this is just TOO vague.

**Comment by ATH1909** [comment]
> Delta's still got an absurd amount of free stuff. The free sec locker in maint, open for theft. The auxiliary engineering room has several insulated gloves, multitools and inducers behind normal walls. The thousands of credits in the abandoned bar as well as the multiple pairs of shades between it and the abandoned gambling den, along with some syndicate cards. And the abandoned arcade has a 2 tc weapon (though it's lacking riot darts, but given it's proximity to cargo, that's not at all any difficulty for a cargo tech or the QM to resolve).
> 
> There is quite a lot more free gamer gear in maint, and frankly, you're gonna have your work cut out for you if you want to genuinely attempt to audit that shit.

You FOOL, now look at what you've done! Now the shitty changes like the drape removals are gonna be merged because they've been bundled with sunglasses removals!

**Comment by Doosteroni** [comment]
I’ve been blocked for 30 days, watching and waiting silently. My sentence has been served, and I return to inform you that this sucks

**Comment by ATH1909** [comment]
> I’ve been blocked for 30 days, watching and waiting silently. My sentence has been served, and I return to inform you that this sucks

based

**Comment by OmegaShoots** [comment]
this is honestly quite bad and I'm not sure what you think you're accomplishing with a PR like this
how does this improve the game exactly?

**Comment by spessbro** [comment]
> this is honestly quite bad and I'm not sure what you think you're accomplishing with a PR like this
> how does this improve the game exactly?

have you read the pr itself, the point is to remove static loot from maint to make maint more interesting

**Comment by msgerbs** [comment]
> 
> 
> > this is honestly quite bad and I'm not sure what you think you're accomplishing with a PR like this
> > how does this improve the game exactly?
> 
> have you read the pr itself, the point is to remove static loot from maint to make maint more interesting

Then you should be adding this stuff into the random loot pool so random loot is actually interesting. I don't know that I see how this makes things more interesting though... Why would you not expect to find surgical drakes in a surgery ward for example? As long as the layout of maint itself is static, some loot should be static to match it.

**Comment by ExcessiveUseOfCobblestone** [comment]
> 
> 
> this is honestly quite bad and I'm not sure what you think you're accomplishing with a PR like this
> how does this improve the game exactly?

oh fuck off, he has an explanation in his first post (you may disagree with the rationale) while you can only say "quite bad".

**Comment by ATH1909** [comment]
> > this is honestly quite bad and I'm not sure what you think you're accomplishing with a PR like this
> > how does this improve the game exactly?
> 
> oh fuck off, he has an explanation in his first post (you may disagree with the rationale) while you can only say "quite bad".

image

image

**Comment by ATH1909** [comment]
> > this is honestly quite bad and I'm not sure what you think you're accomplishing with a PR like this
> > how does this improve the game exactly?
> 
> oh fuck off, he has an explanation in his first post (you may disagree with the rationale) while you can only say "quite bad".

He questioned how this improves the game, which the opening post of the PR didn't adequately explain:

image

These are the only two actual reasons why this PR could improve the game that are listed in the OP. The first one is nonsense- for example, those surgical drapes he was talking about are found IN A ROOM DEDICATED TO SURGERY.

The second one argues that if we make maint loot more random, it'll lessen the efficacy of rushing for static maint loot drops. The problem here is that he hasn't really added the stuff he's been removing back into the random loot pool, and some of the things he's removing would make more sense as static spawns (or at least random ones from special shallow pools) in specific locations (like those aforementioned drapes) than as random ones.

In general, this PR seems to be aiming to rip out almost everything that's worth your time out of Maintenance, and if nothing in there is worth your time, why would you ever explore Maintenance (except to hunt valids)? Omega was asking why gutting a "department" like this is needed, and we still don't have a solid reason other than "I don't think these things fit here, even though they clearly do if you look at their surroundings" and "assistants/maint rushers bad" yet (without even any reasons as to WHY maint rushers are bad!).

**Comment by cacogen** [comment]
What an arbitrary and misguided pull request. I was thinking about Delta's maint this morning and how it was fun the coders had neglected to remove yet. And now, here we are.

**Review by kriskog — CHANGES_REQUESTED** [review]
Im fine with everything here except for the mechfab change. Other maps have a prebuilt one, and its also of the maint variety that starts unlinked.

I wouldnt mind seeing more lootspawns added around either, delta has always felt a bit bare on the random stuff. Thats not a requirement though.

Also, expand the changelog slightly. You dont need to include every change, but give a general idea of whats changed.
i.e. "Removed maint static spawns from all stations." or something like that.

**Comment by ExcessiveUseOfCobblestone** [comment]
@ATH1909 i think you have read further into that post than you needed to.

Even those examples, regardless of how you felt about them, is much more constructive than "honestly quite bad". Hell you managed to write at least several sentences with those, id like to see you do the same had he said "its honestly a good change".

**Comment by cacogen** [comment]
Just close this.

**Comment by msgerbs** [comment]
> 
> 
> @ATH1909 i think you have read further into that post than you needed to.
> 
> Even those examples, regardless of how you felt about them, is much more constructive than "honestly quite bad". Hell you managed to write at least several sentences with those, id like to see you do the same had he said "its honestly a good change".

Are we going to ignore the many comments that have given specific reasons why this is bad which have been summarily ignored?

**Comment by kriskog** [comment]
Actually just waiting for @spessbro to readd the mechfab and I'll merge it. You guys are severely overreacting to this PR, most of the stuff getting removed should have been a while ago, for instance see maint sunglasses and insuls removal on meta over a year ago.

**Comment by msgerbs** [comment]
Ok but this removes a ton of stuff that isn't even gamer gear and makes thematic sense to be where it is, purely because "dynamic loot" but then doesn't even add it to the dynamic pool. Maint is mostly static, like it or not, except for a bunch of meaningless trash loot nobody ever even looks at. Removing actually meaningful and thematically correct static look just because it's static is not valid reasoning.

I don't care about removing insuls or other gamer gear. But this removes a bunch of that doesn't fit that description for no good reason.

**Comment by kriskog** [comment]
What ton of stuff are you referring to?

**Comment by msgerbs** [comment]
You know what, never mind, it looks like most of it got put back since I first looked at this.

**Comment by cacogen** [comment]
I guess contrarianism is contagious

**Comment by BigGrease** [comment]
a poor idea

**Comment by ATH1909** [comment]
https://www.google.com/url?sa=i&url=https%3A%2F%2Fjethrotull.proboards.com%2Fthread%2F3009%2F2017-ian-anderson&psig=AOvVaw0vgkniGTDlCDg_K7evkTiB&ust=1605337384924000&source=images&cd=vfe&ved=0CAIQjRxqFwoTCNj26ff5_uwCFQAAAAAdAAAAABAg

**Comment by EOBGames** [comment]
Don't you people have something better to be doing than clogging up my inbox with pointless comments post merge?

**Comment by spessbro** [comment]
hehe i fill eob's inbox :)
